src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[NUMBER_OF_BALLS];
+        Lottery lottery = new Lottery();
+        for (Ball ball : balls) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
```suggestion
            System.out.println(ball);
```
toString is called by default